testing/src/main/java/io/grpc/internal/testing/TestUtils.java

```diff
@@ -180,6 +180,12 @@ public static void installConscryptIfAvailable() {
       conscryptInstallAttempted = true;
       return;
     }
+    // Concrypt native libraries are not available for ARM architectures.
+    String osArch = System.getProperty("os.arch", "");
+    if (osArch.startsWith("aarch") || osArch.startsWith("arm")) {
```

Comment:
qq: could this negatively affect something when testing on android?

Reply:
This method isn't used on Android. Android Conscrypt has to be set up quite different ways.